web3/_utils/module_testing/shh_module.py

```diff
@@ -320,13 +321,26 @@ def test_shh_remove_filter(self, web3):
         topic = '0x13370000'
         shh_filter = web3.parity.shh.newMessageFilter({'decryptWith': None, 'topics': [topic]})
 
-        assert web3.parity.shh.post({
-            'payload': payload,
-            'topics': [topic],
-            'to': {'public': receiver_pub},
-            'priority': 500,
-            'ttl': 400,
-        })
+        try:
+            web3.parity.shh.post({
```

Comment:
for both changes: accepting a failure due to falking, then requiring a failure in in the `with pytest.raises(...)` in the next block doesn't seem very smooth. I don't know if the code after the try/except requires this code to be successful for it to pass.

If the the rest of the test can work after this code has failed, then you could do something like this to simplify things and only run once:
```python
try:
   ...
except ValueError as e:
   # :(
   if str(e) != "Pow too low to compete with other messages":
       raise
```